aws/sqs/sqs.go

```diff
@@ -231,7 +231,11 @@ func (s *SQS) SendNBatch(ctx context.Context, queueURL string, bodies []string)
 		times     = bodiesLen / maxlen
 	)
 	for i := 0; i <= times; i++ {
-		var bodies_batch = bodies[maxlen*i : maxlen*(i+1)]
+		batch_end := maxlen * (i + 1)
+		if maxlen*(i+1) > bodiesLen {
+			batch_end = bodiesLen
+		}
+		var bodies_batch = bodies[maxlen*i : batch_end]
```

Comment:
Doesn't seem to be ideal when the size is maxlen*N :

https://go.dev/play/p/r58PoXnoU5I

Creates an empty slice in the last loop.